test/libsolidity/syntaxTests/abiEncoder/v1_accessing_public_state_variable_via_v2_type.sol

```diff
@@ -1,3 +1,4 @@
+pragma abicoder v1;
```

Comment:
The filename is `via_v2_type`?

Reply:
yes, v1 accessing via v2: https://github.com/ethereum/solidity/pull/10336/files#diff-15e4c458a3d53f26ef929202976e3ce97564f5e431504c788ec1c1be3a3370c0R11